### PR TITLE
remove literal keys

### PR DESCRIPTION
**Pull request by imeckler**
This PR removes all literal keys

### CODE REVIEW

File: src/lib/consensus/proof_of_signature.ml [hunk 1]
```diff
@@ -12,8 +12,8 @@ module Global_public_key = struct
   global_signer_real]
 
   let compressed =
-    Public_key.Compressed.of_base64_exn
-      "KBWuaAm5Sl5jH/dlpiTKQeUUsty/4Rq6Xz2Py2Y2i/VweJmDHwUAAAAB"
+    Snark_params.Tick.Inner_curve.one
+    |> Non_zero_curve_point.of_inner_curve_exn |> Public_key.compress
```
Comment (bkase) [inline src/lib/consensus/proof_of_signature.ml]: This affects our secret gobal public key in the keybase repo too. Can you update that as required?

File: src/lib/global_signer_private_key/global_signer_private_key.ml [hunk 2]
```diff
@@ -1,10 +1,6 @@
-open Signature_lib
-
 (* This file should be replaced with the production version when building
    for production *)
```
Comment (bkase) [inline src/lib/global_signer_private_key/global_signer_private_key.ml]: As mentioned above, can you update the keybase repo?

**Comment by nholland94** [comment]
Let's talk about this in person some. This breaks some things I did to allow us to configure the testnet, and I don't see any real reason why they need to be done.

**Comment by nholland94** [comment]
Reminder as per our conversation offline: please add a json file build artifact to make the generated keys available to deployment scripts.

File: src/lib/consensus/proof_of_signature.ml (hunk 1 above)
Comment (imeckler) [reply src/lib/consensus/proof_of_signature.ml]: Oh I thought we weren't using that anymore?

Comment (jkrauska) [reply src/lib/consensus/proof_of_signature.ml]: Can you help document the technique required to seed the keys with this code change?

**Comment by imeckler** [comment]
added the JSON file -- can you approve @nholland94 ?

**Pull request merged**